### ISSUE TITLE
Please help putting organization in community hands

### ISSUE THREAD

**Issue by mbrgm** [neo-layout/neo-layout#8]
Dear @neo-layout organization owner,

the Neo layout community would like to put this organization in community hands, so we can share efforts to work on the project. If you are still active, could you please help us to contact you, so we can discuss on the subject? If you're no longer active, we would like to contact GitHub support in order to gain access to the organization.

Kind regards

**Comment by poelzi** [comment]
Sure thing. I ran the svn->git sync for many years because I wanted to have a fork with small changes. I never was a fan of SVN. I invited Ben as an owner (through email), but since I did not find your name on the mailinglist I will not yet grant you access.

**Comment by mbrgm** [comment]
@poelzi Fair enough -- I've not been very active in the community until now. I was just putting into action what we discussed in the #neo IRC channel yesterday. So I'm happy if people, who are currently active, somehow in charge and relevant to the community, are added, so the organization can be actively maintained.

**Comment by qwertfisch** [comment]
As the current Trac system has been disabled by Ben, I am working on a Gitea-based fresh start. So the repository would be hosted on a dedicated website, as before. But as many interested people start looking on Github, it would be great to have this repository (because of its name) in the hands of „the community“, at least some which are still active.

@poelzi I would like to be one of the responsible people. My activity is on the IRC channel, and I have also commited some parts to the SVN repository around 2012 (same username). For authentication purposes you might send me an e-mail to the same address as used in the repository – but I cannot see here which one I used (was the data not imported?), and the SVN repo to check has been disabled.
